src/controllers/profileController.ts

```diff
@@ -57,26 +57,28 @@ export default class ProfileController {
             workEmail
           );
           return errorResponseOrSuccessResponse.map(response => {
-            const emailText = localeIt.profileController.editProfile.notificationEmail.content.replace(
-              "%s",
-              user.givenName
-            );
-            // tslint:disable-next-line:no-floating-promises
-            this.emailService
-              .send({
-                html: emailText,
-                subject:
-                  localeIt.profileController.editProfile.notificationEmail
-                    .subject,
-                text: emailText,
-                to: workEmail
-              })
-              .catch(error =>
-                log.error(
-                  "Failed to send email notification for work email change. %s",
-                  error
-                )
+            if (response.value.work_email !== response.value.email) {
+              const emailText = localeIt.profileController.editProfile.notificationEmail.content.replace(
```

Comment:
if we plan to use this kind of templating a lot probably it's worth to introduce some localization framework to have sprintf like methos, ie:

https://www.i18next.com/